### PR TITLE
Using 'nameof' operator instead of magic strings

### PR DESCRIPTION
**Pull request by hishamco**
Using the "nameof" operator instead of magic string will help us if the variable change in the future for whatever the reason

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@hishamco__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft Open Technologies, Inc. and real humans are currently evaluating your PR.</p>

TTYL, MSOTBOT;

File: test/Microsoft.AspNet.Identity.InMemory.Test/InMemoryUserStore.cs [hunk 1]
```diff
@@ -378,7 +378,7 @@ public Task<IList<TUser>> GetUsersForClaimAsync(Claim claim, CancellationToken c
         {
             if (claim == null)
             {
-                throw new ArgumentNullException("claim");
+                throw new ArgumentNullException(namepf(claim));
```
Comment (HaoK) [inline test/Microsoft.AspNet.Identity.InMemory.Test/InMemoryUserStore.cs]: typo?

Comment (divega) [inline test/Microsoft.AspNet.Identity.InMemory.Test/InMemoryUserStore.cs]: Always worth running the build before sending a PR  :smile:

Comment (hishamco) [inline test/Microsoft.AspNet.Identity.InMemory.Test/InMemoryUserStore.cs]: Sorry for the typo, I will update it now

File: test/Microsoft.AspNet.Identity.InMemory.Test/InMemoryUserStore.cs [hunk 2]
```diff
@@ -368,7 +368,7 @@ public Task<IList<TUser>> GetUsersInRoleAsync(string roleName, CancellationToken
         {
             if (String.IsNullOrEmpty(roleName))
             {
-                throw new ArgumentNullException("role");
+                throw new ArgumentNullException(nameof(roleName));
```
Comment (enghch) [inline test/Microsoft.AspNet.Identity.InMemory.Test/InMemoryUserStore.cs]: Won't this change the result from "role" to "roleName"? Is that the right thing?

Comment (Eilon) [inline test/Microsoft.AspNet.Identity.InMemory.Test/InMemoryUserStore.cs]: @enghch yes and I think that was a bug! Which is part of the reason to use `nameof()` :smile:

Comment (hishamco) [inline test/Microsoft.AspNet.Identity.InMemory.Test/InMemoryUserStore.cs]: @Eilon Is there a way to squash the commits without affect the PR?

Comment (HaoK) [inline test/Microsoft.AspNet.Identity.InMemory.Test/InMemoryUserStore.cs]: Just squash on merge always works

Comment (hishamco) [inline test/Microsoft.AspNet.Identity.InMemory.Test/InMemoryUserStore.cs]: @HaoK I did it one time back, but the PR closed

**Comment by hishamco** [comment]
The commits has been squashed

**Comment by HaoK** [comment]
Don't worry about multiple commits, just add new commits to this PR, when I do the merge, I will squash them for you

**Comment by HaoK** [comment]
Looks good! :shipit:

**Comment by hishamco** [comment]
@Eilon Can you merge this fix like the other repos?

**Comment by hishamco** [comment]
Thanks

**Pull request merged**